NAPS2.Lib.WinForms/EtoForms/WinForms/WinFormsListView.cs

```diff
@@ -82,13 +87,38 @@ private void CustomRenderItem(object? sender, DrawListViewItemEventArgs e)
         }
         else
         {
-            height = ImageSize;
+            if (_config.Get(c => c.EnableThumbnailText))
+                height = ImageSize - 12;
+            else
+                height = ImageSize;
             width = (int) Math.Round(height * (image.Width / (double) image.Height));
         }
         var x = e.Bounds.Left + (e.Bounds.Width - width) / 2;
         var y = e.Bounds.Top + (e.Bounds.Height - height) / 2;
         e.Graphics.DrawImage(image, new Rectangle(x, y, width, height));
+        if (_config.Get(c => c.EnableThumbnailText)) 
+        { 
+            // Draw the text below the image
+            //var draw = _config.Get(c => c.EnableThumbnailText);
+            // Create string to draw.
+            String drawString = (e.ItemIndex+1).ToString() + " / "+_view.Items.Count.ToString(); 
+
+            // Create font and brush.
+            Font drawFont = new Font("Arial", 10);
+            SolidBrush drawBrush = new SolidBrush(Color.Black);
+
+            // Create point 
+            float x1 = x + width/2;
```

Comment:
Not sure what IDE you're using but does it have an autoformat feature you can use? Some of the spacing is off (e.g. `width/2` should be `width / 2`)

Reply:
Try pressing Ctrl+K then Ctrl+D and that should autoformat the current file.